main.py

```
@@ -1,69 +1,307 @@
+    ]
+)
 
-logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger("markdown_scraper")
 
 
 class MarkdownScraper:
-    def __init__(self):
+    """Scrapes websites and converts content to markdown with chunking support."""
+    
+    def __init__(self, 
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)